.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
+
+test_patterns = ["test/**"]
+
+[[analyzers]]
+name = "python"
+enabled = true
+
+  [analyzers.meta]
+  runtime_version = "3.x.x"
```

Comment:
Are we going to use this?

Reply:
I was going to ask @ankitdobhal  to remove it =/

Reply:
it is not a required parameter. If the code runs on Python 3, it will work without the runtime version. This is because if it is not provided, DeepSource Analyzer assume the value to be 3.x.x. If the code runs on Python 2, you will have to provide the value 2.x.x.

Reply:
Sorry I missed that, I only checked the fixes which all seemed valid to me. My bad; do you want me to remove it @guedou @gpotter2?